web/src/features/panels/zone/ZoneDetails.tsx

```diff
@@ -108,7 +112,17 @@ export default function ZoneDetails(): JSX.Element {
               displayByEmissions={displayByEmissions}
             />
           )}
-          <Attribution data={data} zoneId={zoneId} />
+          <Attribution zoneId={zoneId} />
+          {isMobile && (
+            <Button
+              backgroundClasses="mt-3"
+              icon={<MdOutlineCloudDownload size={20} />}
+              href="https://electricitymaps.com/?utm_source=app.electricitymaps.com&utm_medium=referral&utm_campaign=country_panel"
+            >
+              {t('header.get-data')}
+            </Button>
+          )}
+          {!isMobile && <div className="p-2" />}
```

Comment:
```suggestion
          <Attribution zoneId={zoneId} />
          {isMobile ? (
            <Button
              backgroundClasses="mt-3"
              icon={<MdOutlineCloudDownload size={20} />}
              href="https://electricitymaps.com/?utm_source=app.electricitymaps.com&utm_medium=referral&utm_campaign=country_panel"
            >
              {t('header.get-data')}
            </Button>
          ) : <div className="p-2" />}
```

Combines it into one conditional and reduces the code a tiny bit.